File: problems/s/stopwatch/stopwatch.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())
    if n % 2 == 0:
        aux_t = 0
        sum_t = 0
        for i in range(n):
            t = int(input())
            if i % 2 == 0:
                aux_t = t
            else:
                sum_t += t - aux_t
        print(sum_t)
    else:
        for i in range(n):
            t = int(input())
        print("still running")


if __name__ == '__main__':
    main()

File: problems/s/stopwatch/stopwatch.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int n, t, aux_t, sum_t;
    if (scanf("%d\n", &n) != 1) {
        printf("Failed to read n!");
    }

    if (n % 2 == 0) {
        aux_t = 0;
        sum_t = 0;
        for (int i=0; i<n; i++) {
            if (scanf("%d\n", &t) != 1) {
                printf("Failed to read t!");
            }
            if (i % 2 == 0) {
                aux_t = t;
            }
            else {
                sum_t += t - aux_t;
            }
        }
        printf("%d", sum_t);
    }
    else {
        for (int i=0; i<n; i++) {
        }
        printf("still running");
    }

    return 0;
}